Frontend/src/cse-ui-kit/container/container.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { StyledContainer, containerProps } from './container-Styled';
+
+
+type Props = {
+  className?: string;
+} & containerProps;
+
+export default function Container({ className}: Props) {
+  return (
+    <div className={className}>
+      <header>
+          <StyledContainer></StyledContainer>
+      </header>      
+    </div>
+  );
+}
```

Comment:
@leannachan Container should take in a Children props, and should be 
```js
<StyledContainer>
  {children}
</StyledContainer>
```
this is so we can actually pass children into the container.